Sources/Plasma/NucleusLib/inc/plPipeline.h

```diff
@@ -138,6 +138,22 @@ class plDisplayMode
     int ColorDepth;
 };
 
+class plDisplayHelper
+{
+public:
+    virtual plDisplayMode DesktopDisplayMode() = 0;
+    virtual std::vector<plDisplayMode> GetSupportedDisplayModes(hsDisplayHndl display, int ColorDepth = 32) = 0;
+    
+    plDisplayHelper() = default;
+    virtual ~plDisplayHelper() = default;
+    
+    static plDisplayHelper* CurrentDisplayHelper() { return _currentDisplayHelper; }
+    void MakeCurrentDisplayHelper() { _currentDisplayHelper = this; }
```

Comment:
The rest of the engine uses `SetInstance()` and `GetInstance()` for singletons. It would be nice to maintain consistency with the already existing patterns.